src/ontology/mondo-edit.obo

```diff
@@ -452064,6 +452064,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/600 {xref="Orphanet:425"} ! APOA1
 created_by: https://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100190
+name: gastric intestinal metaplasia
+def: "Metaplastic changes in the lining of the upper digestive tract." [https://my.clevelandclinic.org/health/diseases/22215-intestinal-metaplasia, PMID:29606921]
+synonym: "GIM" EXACT [PMID:29606921]
+synonym: "intestinal metaplasia" EXACT [NCIT:C8360, PMID:29606921]
+xref: NCIT:C8360 {source="MONDO:equivalentTo"}
```

Comment:
I think you should use MONDO:relatedTo (as it is not equivalent)

Reply:
good point

see this similar example

<img width="1393" alt="image" src="https://user-images.githubusercontent.com/6722114/204930675-b222567a-c66f-449a-ad03-970a7f6fe2f5.png">